README.md

```diff
@@ -42,21 +38,24 @@ If you've installed the C++ Vscode extension you should configure it to format o
 
 Each module has its own tests. e.g. To build and run `ecc` tests:
 
-```
-cmake --build . --parallel --target ecc_tests
```

Comment:
Are we removing `--parallel` everywhere just because it shouldn't be listed as a recommended/default option?

Reply:
`--parallel` doesn't work with presets. I'm pretty sure that's because it's a legacy option and `--jobs N` is supposed to be used; however I didn't try combining presets with `--jobs`. Instead, we set `jobs: 0` on all our presets so they are running at maximum concurrency.

Reply:
Slight correction, it seems that `--parallel` works with presets, but requires a very specific ordering (e.g. you need to specify `--build --preset default --parallel` otherwise the preset isn't picked up correctly). Since we set `jobs: 0` in all the presets, we shouldn't recommend the flag because it just makes the command invocation more complex.